operations/security_and_authentication/Update_default_Air-Cooled_BMC_and_Leaf_switch_SNMP_credentials.md

```diff
@@ -0,0 +1,196 @@
+# Update default Air-Cooled BMC and Leaf switch SNMP credentials
```

Comment:
Capitalization change: "Update Default Air-Cooled BMC and Leaf Switch SNMP Credentials"

Reply:
Should be resolved fixed now.